frontend/common/src/components/UserProfile/ProfileSections/DiversityEquityInclusionSection.tsx

```diff
@@ -68,35 +100,153 @@ const DiversityEquityInclusionSection: React.FunctionComponent<{
+                              src={imageUrl("/", "metis-true.png")}
+                            />
+                          );
+                        case IndigenousCommunity.Other:
+                          return (
+                            <img
+                              data-h2-float="base(right)"
+                              data-h2-height="base(4em)"
+                              alt=""
+                              key="other-true"
+                              src={imageUrl("/", "other-true.png")}
+                            />
+                          );
+                        default:
+                          return null;
+                      }
+                    })}
+                </div>
+              </div>
+            )}
+            {(isWoman || isVisibleMinority || hasDisability) && (
+              <ul
+                data-h2-font-weight="base(700)"
+                data-h2-padding="base(x1, 0, 0, x1)"
+              >
+                {isWoman && (
+                  <li>
+                    {intl.formatMessage(getEmploymentEquityStatement("woman"))}
+                  </li>
+                )}
+                {isVisibleMinority && (
+                  <li>
+                    {intl.formatMessage(
+                      getEmploymentEquityStatement("minority"),
+                    )}
+                  </li>
+                )}{" "}
+                {hasDisability && (
+                  <li>
+                    {intl.formatMessage(
+                      getEmploymentEquityStatement("disability"),
+                    )}
+                  </li>
+                )}
+              </ul>
+            )}
+            <p
+              data-h2-padding="base(x1, 0, 0, 0)"
+              data-h2-color="base:(ia-gray.dark)"
             >
-              {isWoman && (
-                <li>
-                  {intl.formatMessage(getEmploymentEquityStatement("woman"))}
-                </li>
-              )}
-              {isIndigenous && (
-                <li>
-                  {intl.formatMessage(
-                    getEmploymentEquityStatement("indigenous"),
-                  )}
-                </li>
-              )}{" "}
-              {isVisibleMinority && (
-                <li>
-                  {intl.formatMessage(getEmploymentEquityStatement("minority"))}
-                </li>
-              )}{" "}
-              {hasDisability && (
-                <li>
-                  {intl.formatMessage(
-                    getEmploymentEquityStatement("disability"),
-                  )}
-                </li>
-              )}
-            </ul>
+              {intl.formatMessage({
+                defaultMessage:
+                  "You have identified as a member of an <underline>employment equity group.</underline><strong> You can add additional <underline>employment equity groups</underline> to your profile by editing this section if they apply to you.</strong>",
```

Comment:
Should these underlines be links to somewhere?  Maybe to a page about the Employment Equity Act of Canada?

Reply:
> Should these underlines be links to somewhere? Maybe to a page about the Employment Equity Act of Canada?

@BrookeCameron Regarding the text here: 
![image](https://user-images.githubusercontent.com/49166751/216731944-989d0f9e-2dd6-4628-bd95-c2c8c6033684.png)

Reply:
Yeah, Wave doesn't like it either.
![image](https://user-images.githubusercontent.com/8978655/217004085-f8077f03-0ab9-40c6-afb4-0ee647afc458.png)

Reply:
As per Gray, remove underlines.

Reply:
Removed in [0f30298](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5494/commits/0f30298411b477515aec36e3cf5fa50746b07ef5).

Reply:
:+1: